rocAL/rocAL_pybind/examples/PYTHON_UNITTEST_TEST_FILE.sh

```diff
@@ -1,149 +1,122 @@
 #!/bin/bash
+cwd=$(pwd)
 
-if [[ $# -gt 0 ]]; then
-    helpFunction()
-    {
-    echo ""
-    echo "Usage: $0 [-n number_of_gpus] [-d dump_outputs<true/false>] [-b backend<cpu/gpu>] [-p print_tensor<true/false>]"
-    echo -e "\t-n Description of what is the number of gpus to be used"
-    echo -e "\t-d Description of what is the display param"
-    echo -e "\t-p Description of what is the print tensor param"
-    exit 1 # Exit script after printing help
-    }
-
-    while getopts "n:d:b:p:" opt
-    do
-        echo $opt
-        case "$opt" in
-            n ) number_of_gpus="$OPTARG" ;;
-            d ) dump_outputs="$OPTARG" ;;
-            b ) backend="$OPTARG" ;;
-            p ) print_tensor="$OPTARG" ;;
-            ? ) helpFunction ;; # Print helpFunction in case parameter is non-existent
-        esac
-    done
-
-    # Print helpFunction in case parameters are empty
+if [[ $ROCAL_DATA_PATH == "" ]]
+then 
+    echo "Need to export ROCAL_DATA_PATH"
+    exit
+fi
 
-    if [ -z "$backend" ];
-    then
-        backend_arg=no-rocal-gpu
-    else
-        if [[ $backend == "cpu" || $backend == "CPU" ]]; then
-            backend_arg=no-rocal-gpu
-        elif [[ $backend == "gpu" || $backend == "GPU" ]]; then
-            backend_arg=rocal-gpu
-        fi
+# Path to inputs and outputs available in MIVisionX-data
+image_path=${ROCAL_DATA_PATH}/rocal_data/coco/coco_10_img/train_10images_2017/
+coco_detection_path=${ROCAL_DATA_PATH}/rocal_data/coco/coco_10_img/train_10images_2017/
+tf_classification_path=${ROCAL_DATA_PATH}/rocal_data/tf/classification/
+tf_detection_path=${ROCAL_DATA_PATH}/rocal_data/tf/detection/
+caffe_classification_path=${ROCAL_DATA_PATH}/rocal_data/caffe/classification/
+caffe_detection_path=${ROCAL_DATA_PATH}/rocal_data/caffe/detection/
+caffe2_classification_path=${ROCAL_DATA_PATH}/rocal_data/caffe2/classification/
+caffe2_detection_path=${ROCAL_DATA_PATH}/rocal_data/caffe2/detection/
+mxnet_path=${ROCAL_DATA_PATH}/rocal_data/mxnet/
+output_path=./rocal_python_unittest_output_folder_$(date +%Y-%m-%d_%H-%M-%S)/
+golden_output_path=${ROCAL_DATA_PATH}/rocal_data/complete_tensor_golden_output/
+
+display=0
+batch_size=2
+device=0
+width=640 
+height=480
+device_name="host"
+rgb_name=("gray" "rgb")
+rgb=1
+dev_start=0
+dev_end=1
+rgb_start=0
+rgb_end=1
+
+if [ "$#" -gt 0 ]; then 
+    if [ "$1" -eq 0 ]; then # For only HOST backend
+        dev_start=0
```

Comment:
The c++ and the python unit tests run for both backends if its not specified otherwise. 0 corresponds to Host and 1 corresponds to HIP so they run a for loop for switching the backends for dumping the augmentation outputs